content/ko/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,106 @@
+---
+title: IPv4/IPv6 이중 스택
+feature:
+  title: IPv4/IPv6 이중 스택
+  description: >
+    파드와 서비스에 IPv4와 IPv6 주소 할당
+
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+ IPv4/IPv6 이중 스택을 사용하면 {{< glossary_tooltip text="파드" term_id="pod" >}} 와 {{< glossary_tooltip text="서비스" term_id="service" >}} 에 IPv4와 IPv6 주소를 모두 할당 할 수 있다.
+
+만약 쿠버네티스 클러스터에서 IPv4/IPv6 이중 스택 네트워킹을 활성화하면, 클러스터는 IPv4와 IPv6 주소의 동시 할당을 지원하게 된다.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 지원되는 기능
+
+쿠버네티스 클러스터에서 IPv4/IPv6 이중 스택을 활성화하면 다음의 기능을 제공한다.
+
+   * 이중 스택 파드 네트워킹(단일 IPv4와 IPv6 주소를 파드에 할당)
+   * 서비스에 IPv4와 IPv6이 가능하게 된다(각 서비스는 단일 주소 패밀리이어야 한다.)
+   * Kubenet 다중 주소 패밀리 지원(IPv4와 IPv6)
+   * IPv4와 IPv6 인터페이스를 통한 파드 오프(off) 클러스터 송신 라우팅(예: 인터넷)
+
+## 필수 구성 요소
+
+IPv4/IPv6 이중 스택 쿠버네티스 클러스터를 활용하려면 다음의 필수 구성 요소가 필요하다.
+
+   * 쿠버네티스 1.16 또는 이후 버전
+   * 이중 스택 네트워킹을 위한 공급자의 지원(클라우드 공급자 또는 다른 방식으로 쿠버네티스 노드에 라우팅 가능한 IPv4/IPv6 네트워크 인터페이스를 제공할 수 있어야 한다)
```

Comment:
> origin
>    * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)

```suggestion
   * 이중 스택 네트워킹을 위한 공급자의 지원(클라우드 공급자 또는 다른 방식으로 쿠버네티스 노드에 라우팅 가능한 IPv4/IPv6 네트워크 인터페이스를 제공할 수 있어야 한다.)
```

- [ ] `.`이 누락되었습니다.